README.md

```diff
@@ -1,25 +1,20 @@
-Gocdb
-=====
-Grid Operations Configuration Management Database. A Repository, Portal and 
-REST style API for managing Grid and Cloud topology objects including; projects, 
-administrative domains, sites, services, service-endpoints, service-groups, 
-downtimes, users, roles and business rules.  The View and Controller layers 
-(i.e. the 'VC' in 'MVC') are inherited legacy from
-v4 and are scheduled to be re-implement using a modern MVC framework. 
+# GOCDB
 
-<img src="htdocs/landing/egi_logo.jpg" height="25" style="float: left;" />
-GOCDB is an EGI service provided by <a class="Licence_Link" href="http://www.stfc.ac.uk/">
-STFC</a>, co-funded by <a class="Licence_Link" href="http://www.egi.eu">EGI.eu</a> and 
-<a href="http://go.egi.eu/eng">EGI-Engage</a>. Licensed under the 
-<a class="Licence_Link" href="http://www.apache.org/licenses/LICENSE-2.0">Apache 2 License</a>
+[![Build Status](https://travis-ci.org/GOCDB/gocdb.svg?branch=dev)](https://travis-ci.org/GOCDB/gocdb)
 
+Grid Operations Configuration Management Database. A Repository, Portal and REST style API for managing Grid and Cloud topology objects including; projects, administrative domains, sites, services, service-endpoints, service-groups, downtimes, users, roles and business rules.
 
-Documentation 
-=============
-* [GOCDB Executive Summary](https://wiki.egi.eu/w/images/d/d3/GOCDB5_Grid_Topology_Information_System.pdf)
-* [EGI GOCDB wiki landing page](https://wiki.egi.eu/wiki/GOCDB) 
-* [GOCDB Documentation index](https://wiki.egi.eu/wiki/GOCDB_Documentation_Index)
-* [User docs](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation) 
-* INSTALL.md for installation 
+<img alt="STFC logo" src="htdocs/web_portal/img/STFCMediumColourTrans_crop.png" height=57/> <img alt="EU flag" src="htdocs/web_portal/img/eu_flag_yellow_low_150.png" height=57 /> <img alt="EGI logo" src="htdocs/web_portal/img/egi_logo.jpg" height=57 /> <img alt="EOSC-hub logo" src="htdocs/web_portal/img/eosc-hub-v-web_150.png" height=57 />
+
+GOCDB is provided by <a class="Licence_Link" href="https://stfc.ukri.org/">
```

Comment:
Fairly sure that the class won't be do anything here, so the markdown syntax would be more readable:
`[STFC](https://stfc.ukri.org/)`

Reply:
Yup, but I just left it as it was. Can fix it as I'm here anyway.